contracts/finality/src/utils.rs

```diff
@@ -43,6 +44,35 @@ pub fn query_finality_provider(
     Ok(res)
 }
 
+/// Validates that a consumer ID has a valid format (non-empty, valid characters, etc.)
+pub fn validate_consumer_id_format(consumer_id: &str) -> Result<(), ContractError> {
+    if consumer_id.is_empty() {
```

Comment:
Imo this function should be put under https://github.com/babylonlabs-io/rollup-bsn-contracts/blob/c0d83dfee8a6f06599248715df88d46934e1bd37/contracts/finality/src/exec/admin.rs